test/integration/acl-oidc-test.js

```diff
@@ -545,6 +545,16 @@ describe('ACL with WebID+OIDC over HTTP', function () {
         done()
       })
     })
+    it('user2 should not be able able to post an acl file', function (done) {
```

Comment:
This test should actuallly be like this:
(do a POST with a Slug Header, not PUT)

```javascript
it('user2 should not be able able to post an acl file', function (done) {
      var options = createOptions('/append-acl/', 'user2', 'text/turtle')
      options.headers.Slug = 'some.acl'
      options.body = '<a> <b> <c> .\n'
      request.post(options, function (error, response, body) {
        assert.equal(error, null)
        assert.equal(response.statusCode, 403)
        assert.equal(response.statusMessage, 'User Unauthorized')
        done()
      })
    })
```